src/components/AdjustVaultInstructions.tsx

```diff
@@ -0,0 +1,18 @@
+export const AdjustVaultInstructions = () => (
+  <div className="bg-white shadow-[0_22px_34px_rgba(116,116,116,0.25)] rounded-[10px] px-7 py-4">
+    <h3 className="text-[#484848] font-bold font-serif">Instructions</h3>
+    <ol className="list-decimal text-[#A3A5B9] text-sm mx-4 my-3 [&>li]:mt-1">
```

Comment:
20 other`A3A5B9` in main. Time to name this one?

Reply:
I'd like to do that as part of https://github.com/Agoric/dapp-inter/issues/46, which is second in my queue currently behind https://github.com/Agoric/dapp-inter/issues/127